six/modules/c++/six.sidd/include/six/sidd/CompressedSIDDByteProvider.h

```diff
@@ -57,6 +57,7 @@ class CompressedSIDDByteProvider : public six::CompressedByteProvider
      * \param bytesPerBlock A vector for each image segment. Each inner vector
      *        contains the compressed size for each block in the segment,
      *        in bytes.
+     * \param compressionRatio Ratio used for J2K compression
```

Comment:
Not sure if you had finished the code up and were ready for review or not, but need more comments here to explain what this is a ratio of